sickbeard/providers/torrent/html/scc.py

```diff
@@ -46,32 +46,34 @@ def __init__(self):
         # URLs
         self.url = 'https://sceneaccess.eu'
         self.urls = {
-            'base_url': self.url,
             'login': urljoin(self.url, 'login'),
-            'detail': urljoin(self.url, 'details?id=%s'),
-            'search': urljoin(self.url, 'all?search=%s&method=1&%s'),
-            'download': urljoin(self.url, '%s')
+            'search': urljoin(self.url, 'all?search={string}&method=1&{cats}'),
         }
 
         # Proper Strings
+        self.proper_strings = ['PROPER', 'REPACK', 'REAL']
 
         # Miscellaneous Options
         self.categories = {
-            'Season': 'c26=26&c44=44&c45=45',  # Archive, non-scene HD, non-scene SD; need to include non-scene because WEB-DL packs get added to those categories
-            'Episode': 'c17=17&c27=27&c33=33&c34=34&c44=44&c45=45',  # TV HD, TV SD, non-scene HD, non-scene SD, foreign XviD, foreign x264
-            'RSS': 'c17=17&c26=26&c27=27&c33=33&c34=34&c44=44&c45=45'  # Season + Episode
+            # Archive, non-scene HD, non-scene SD;
+            # need to include non-scene because WEB-DL packs get added to those categories
+            'Season': 'c26=26&c44=44&c45=45',
+            # TV HD, TV SD, non-scene HD, non-scene SD, foreign XviD, foreign x264
+            'Episode': 'c17=17&c27=27&c33=33&c34=34&c44=44&c45=45',
+            # Season + Episode
+            'RSS': 'c17=17&c26=26&c27=27&c33=33&c34=34&c44=44&c45=45',
         }
 
         # Torrent Stats
         self.minseed = None
         self.minleech = None
 
         # Cache
-        self.cache = tvcache.TVCache(self)  # only poll SCC every 20 minutes max
+        self.cache = tvcache.TVCache(self, min_time=20)
```

Comment:
@medariox default is 10 afaik. are you sure you want to change to 20 to this provider?